### PR TITLE
enhancement: Add identifier type name alongside value on patient-banner

### PR DESCRIPTION
**Pull request by donaldkibet**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide).
- [ ] My work includes tests or is validated by existing tests.

## Summary
- Enhancement add identifier type name alongside the value to enable users know which identifier they are looking at. Screenshot for reference
- Remove grey underscore on active visit tag 

## Screenshots
Screenshot from 2022-09-14 21-59-22

### CODE REVIEW

**Comment by makombe** [comment]
@donaldkibet  thanks for this, I think you can rearrange them display key then value. where key is the identifiertype eg `Openmrs Id: MGJAUL`

**Review by CynthiaKamau — APPROVED** [review]
Thanks Donald

File: packages/esm-patient-banner-app/src/banner/patient-banner.scss [hunk 1]
```diff
@@ -1,6 +1,8 @@
+@use '@carbon/styles/scss/spacing';
 @use '@carbon/styles/scss/type';
 @import '~@openmrs/esm-styleguide/src/vars';
 
+
 .container {
   border-top: 0.063rem solid $ui-03;
```
Comment (CynthiaKamau) [inline packages/esm-patient-banner-app/src/banner/patient-banner.scss:7]: Why not use carbon spacing

Comment (denniskigen) [reply packages/esm-patient-banner-app/src/banner/patient-banner.scss:7]: Counterpoint: I actually reckon we shouldn't be using the Carbon spacing tokens - they're not parseable at a glance in my opinion and I reckon we should prefer using em/rem-based spacing directly.

**Pull request merged**